components/suggestion/search_channel_with_permissions_provider.jsx

```diff
@@ -44,15 +41,15 @@ class SearchChannelWithPermissionsSuggestion extends Suggestion {
         let icon = null;
         if (channelIsArchived) {
             icon = (
-                <ArchiveIcon className='icon icon__archive'/>
+                <i className='icon icon--no-spacing icon-archive-outline'/>
```

Comment:
Are we going back to font-awesome with this icons? Shouldn't we use the icon widgets for this, because if not in the future we have to modify each one of this, instead of modifying only the component one.

Reply:
@jespino No, this is the mattermost icon font. From now on, we'll try to use the mattermost icon font in the webapp instead of the svgs.

Reply:
Great, still I think have value to wrap it in a component. But that is another subject. Thanks for the explanation.